utils/consts.go

```diff
@@ -2384,17 +2384,17 @@ const (
 	// EEs
 	OptsEEsVerbose = "*eesVerbose"
 	// EEs Elasticsearch options
-	Index               = "index"
-	IfPrimaryTerm       = "if_primary_term"
-	IfSeqNo             = "if_seq_no"
-	OpType              = "op_type"
-	Pipeline            = "pipeline"
-	RequireAlias        = "require_alias"
-	Routing             = "routing"
-	Timeout             = "timeout"
-	VersionLow          = "version"
-	VersionType         = "version_type"
-	WaitForActiveShards = "wait_for_active_shards"
+	OptsIndex               = "index"
```

Comment:
Since these options are elastic search related, please rename all the constants used there, ie: ElsIndex, ElsIfPrimaryTerm